Applications/LLMEval/ViewModels/DeviceStat.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import LLM
+import MLX
+
+class DeviceStat: ObservableObject {
```

Comment:
Maybe use `@Observable` (like in ContentView)?  The minimum SDK for the apps has these macros and they are more forward looking.

Reply:
Thanks for the comment, I have updated the DeviceStat to use @Observable macro